contrib/util/ubuntu_package_scripts/development/postinst

```diff
@@ -222,7 +222,9 @@ case "$1" in
 
       # Set file and directory permissions
       chmod 666 $SITEDIR/sqlconf.php
-      chown $WEB_GROUP.$WEB_USER $OPENEMR/interface/modules/zend_modules/config/application.config.php
+      chown $WEB_GROUP.$WEB_USER $SITEDIR/
```

Comment:
Making the entire sitedir writable is not a good idea out of the box. Then a user can modify the config scripts in there out of the box (can directly inject php that can then be easily run).